### ISSUE TITLE
Bootstrap 5 theme incoming any time soon? 

### ISSUE THREAD

**Issue by martinbarilik** [ttskch/select2-bootstrap4-theme#66]
Hi there, i am rewriting the an old web app. I want it to be the latest at everyting, today i changed bootstrap 4 to bootstrap 5 and using select, some theme is messy. 

Can we expect bootstrap 5 theme any time soon ?

**Comment by tagliala** [comment]
Hi,

You can find a Bootstrap 5 compatible theme here: https://github.com/tagliala/select2-bootstrap4-theme/tree/feature/bootstrap-5

It is based on my work done on #67, so it requires Select2 4.1

(and don't upgrade jQuery to 3.6.0, as per select2/select2#5993)

![image](https://user-images.githubusercontent.com/556268/111070170-9acb2280-84d0-11eb-96dd-785258d7b896.png)

![image](https://user-images.githubusercontent.com/556268/111070197-c1895900-84d0-11eb-916f-cac9811b54a5.png)

**Comment by martinbarilik** [comment]
Even tho it is compatible with BS5, it does not support bootstrap's form-control-sm classes ( small selectboxes ). More [here](https://getbootstrap.com/docs/5.0/forms/form-control/#sizing).

**Comment by tagliala** [comment]
> it does not support bootstrap's form-control-sm classes ( small selectboxes ).

Yes, this theme does not support small controls neither in BS4. Ref: #40

**Comment by tagliala** [comment]
@martinbarilik just saw this in the Select2 repo: https://github.com/apalfrey/select2-bootstrap-5-theme

Demo doesn't work, but it looks like a complete theme with support for small and large classes

**Comment by martinbarilik** [comment]
Holy Molly, it's recent. Thanks, last i checked it wasn't any ( 2 weeks ago ).

**Comment by MarcPinnell** [comment]
@tagliala - Were you able to make that BS5 theme work for the small dropdown size?

**Comment by tagliala** [comment]
@MarcPinnell please take a look at https://apalfrey.github.io/select2-bootstrap-5-theme/

**Comment by MarcPinnell** [comment]
I was never able to get that working. I did find a BS4 theme that I managed to get working yesterday. Not ideal, but it works.
